### PR TITLE
Add github actions worklow file

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: .github/workflow/ci.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      - master
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
```
Comment (dblenkus) [inline .github/workflow/ci.yml:15]: Would it make sense to use `ubuntu-20.04`?

File: .github/workflow/ci.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        PGPASSWORD: resolwe
+        RESOLWE_POSTGRESQL_PORT: ${{ job.services.postgres.ports[5432] }}
+    - name: Upload coverage to Codecov
+      if: matrix.tox-env == 'py36' || matrix.tox-env == 'py37' || matrix.tox-env == 'py38'
+      uses: codecov/codecov-action@v1
+      with:
+        file: .coverage
+      
+  build:
+    runs-on: ubuntu-18.04
```
Comment (dblenkus) [inline .github/workflow/ci.yml:76]: `ubuntu-20.04`

File: docs/CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Added
+-----
+- Github actions workflow file
```
Comment (dblenkus) [inline docs/CHANGELOG.rst:15]: This is not a user-facing change, so you don't need to add a changelog.

File: .github/workflow/ci.yml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip tox
+    - name: Run Tests
+      run: |
+        RESOLWE_DOCKER_DISABLE_SECCOMP=1 tox -r -e ${{ matrix.tox-env }} --skip-missing-interpreters false
```
Comment (dblenkus) [inline .github/workflow/ci.yml:60]: I prefer long flags' names in committed files - `--recreate` instead of `-r`.

Comment (dblenkus) [reply .github/workflow/ci.yml:60]: Also, you can replace `|` in the line above with `>` and break this line into multiple lines as newlines will be ignored by the YAML parser.

**Review by dblenkus — COMMENTED** [review]
Looks good in general.

File: .github/workflow/ci.yml (hunk 1 above)
Comment (gregorjerse) [reply .github/workflow/ci.yml:15]: Sure. I will test it on the new protocol PR soon to see if it works as expected.

File: .github/workflow/ci.yml (hunk 2 above)
Comment (gregorjerse) [reply .github/workflow/ci.yml:76]: OK.

File: docs/CHANGELOG.rst (hunk 3 above)
Comment (gregorjerse) [reply docs/CHANGELOG.rst:15]: Great.

File: .github/workflow/ci.yml (hunk 4 above)
Comment (gregorjerse) [reply .github/workflow/ci.yml:60]: OK, will try that.

**Comment by acopar** [comment]
Update the build badge in README, so we can see when github actions build fails.

**Review by acopar — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        ports: 
+          - 5432/tcp
+      
+      redis:
+        image: redis:alpine
+        ports: 
+          - 6379/tcp
+
+    steps:
+    - uses: actions/checkout@v2
```
Comment (dblenkus) [inline .github/workflows/ci.yml:50]: Add an extra indentation to this level.

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+      - uses: actions/checkout@v1
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install pep517
+        run: python -m pip install --user pep517
+      - name: Build a binary wheel and a source tarball
+        run: python -m pep517.build --source --binary --out-dir dist/ .
+
```
Comment (dblenkus) [inline .github/workflows/ci.yml:94]: Is there a reason for this new line?

**Review by dblenkus — APPROVED** [review]
Looks good, I only have two small comments.

File: .github/workflows/ci.yml (hunk 5 above)
Comment (gregorjerse) [reply .github/workflows/ci.yml]: OK

File: .github/workflows/ci.yml (hunk 6 above)
Comment (gregorjerse) [reply .github/workflows/ci.yml]: Nope.

**Pull request merged**